src/app/components/webauthn/login.tsx

```diff
@@ -24,12 +24,12 @@ async function webAuthnLogin(challenge: Uint8Array) {
 
 const login = async () => {
     let {challenge, authenticator_data, client_data_json, signatureRaw} =
-        await webAuthnLogin(new Uint8Array([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,]));
+        await webAuthnLogin(new Uint8Array([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]));
 
-    const challenge_str = `challenge = [${challenge}}]`;
-    const client_data_json_str = `client_data_json = [${client_data_json}]`;
-    const authenticator_data_str = `authenticator_data = [${authenticator_data}]`;
-    const signature_str = `signature = [${new Uint8Array(signatureRaw)}]`;
+    const challenge_str = `let challenge = [${challenge}];`;
+    const client_data_json_str = `let client_data_json = [${client_data_json}];`;
+    const authenticator_data_str = `let authenticator_data = [${authenticator_data}];`;
+    const signature_str = `let signature = [${new Uint8Array(signatureRaw)}];`;
```

Comment:
What kind of dark magic is that?
Why not just `console.log(\`let challenge = [${challenge}];\`)`

Reply:
A this point I wanted to manually check if WebAuth output can be properly checked on Noir contract side. So just for convenience I want to output proper Noir code in one console output for easier copy-paste.